src/Renderer/Shader/Chunk/elevation_pars_vertex.glsl

```diff
@@ -36,7 +37,9 @@
     float getElevation(vec2 uv, sampler2D tex, vec4 offsetScale, Layer layer) {
         uv = uv * offsetScale.zw + offsetScale.xy;
         float d = getElevationMode(uv, tex, layer.mode);
-        if (d < layer.zmin || d > layer.zmax) d = 0.;
+        if (d < layer.zmin) d = layer.zmin;
+        if (d > layer.zmax) d = layer.zmax;
+        if (d == layer.noDataValue) d = 0.;
```

Comment:
there's a `clamp` method in glsl

it seems to me that `d` is an interpolated value,  the filter `nodatavalue` will not work here

Reply:
I changed it with the clamp method.

Reply:
I still want the nodata value to be interpreted as an elevation of zero in the texture. Can we check that together ?